galpy/potential/TwoPowerSphericalPotential.py

```diff
@@ -8,11 +8,14 @@
 ###############################################################################
 import math as m
 import numpy
-from scipy import special, optimize, integrate
+from numpy import sqrt, pi
```

Comment:
``galpy`` coding style is to always explicitly include ``numpy.`` in numpy calls (there is older code that uses ``nu.`` left, but now should be ``numpy.``)

Reply:
done.